### PR TITLE
Add codecov support

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Fixes https://github.com/giantswarm/builder/issues/69

### CODE REVIEW

**Comment by JosephSalisbury** [comment]
No tests yet, though :(

**Comment by JosephSalisbury** [comment]
@giantswarm/cena RFR

File: .travis.yml [hunk 1]
```diff
@@ -13,4 +13,7 @@ script:
 - make ci-lint
 - make ci-test
 - make ci-build
-- ./builder # Check that the binary runs
\ No newline at end of file
+- ./builder # Check that the binary runs
+
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```
Comment (zeisss) [inline .travis.yml]: has anyone looked at this script? It's massive :( not really a fan of this intransparency, but ok.

**Comment by zeisss** [comment]
LGTM

File: .travis.yml (hunk 1 above)
Comment (xh3b4sd) [inline .travis.yml]: The alternative is a even more ugly python script that is broken anyway at the moment. That is why I decided to use the bash uploader. It is also not nice to have this `go.test.sh`, but this is simply the way it works. Important is here the result, and this is good.

**Pull request merged**